### PR TITLE
Update lib injection with lack of uWSGI support

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
### What does this PR do?

Update the lib injection docs about uWSGI.

### Motivation

The Python injection mechanism does not currently support uWSGI so we should call it out to avoid confusion.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by rtrieu — CHANGES_REQUESTED** [review]
Hey @Kyle-Verhoog , we generally do not advertise what we don't support in our docs. We list what we do support. Is there a different way you can frame this?

**Comment by elee0625** [comment]
Hey @rtrieu - we're adding this in explicitly because we've had a few customers ask about uWSGI for the Python tracer, and were hoping to avoid further escalations in the future. Is it okay to make an exception or do you have other recommended wording?

File: content/en/tracing/trace_collection/library_injection.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ To learn more about Kubernetes Admission Controller, read [Kubernetes Admission
 * Datadog [Cluster Agent v7.40+][3] with Datadog Admission Controller enabled. **Note**: In Helm chart v2.35.0 and later, Datadog Admission Controller is activated by default in the Cluster Agent.
 * Applications in Java, JavaScript, or Python deployed on Linux with a supported architecture. Check the [corresponding container registry](#container-registries) for the complete list of supported architectures by language.
 
+<div class="alert alert-warning">Python uWSGI applications are currently not supported.</div>
```
Comment (rtrieu) [inline content/en/tracing/trace_collection/library_injection.md:32]:
```suggestion
**Note:** Python uWSGI applications are not supported.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Hey @elee0625 , let's change it to a note instead and remove "currently" so it doesn't seem like we are promising to support it in the future.

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**